src/main/resources/static/index.html

```diff
@@ -10,5 +10,16 @@
 </head>
 <body>
     <p>Here should show a different number</p>
-    <img src="api/number/">
```

Comment:
are any of changes in index.html needed?

Reply:
nope, deleted

Reply:
I meant changes, not code :D

Reply:
Ok, got it I think when I was trying to make it work I tried to do something in index.html and the changes remained there.

Reply:
ok, so please revert it to code as it is on master branch

Reply:
done